.github/workflows/release_linux_pytorch_wheels.yml

```diff
@@ -30,17 +52,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        config:
-          - { AMDGPU_FAMILIES: gfx94X-dcgpu }
-          - { AMDGPU_FAMILIES: gfx110X-dgpu }
-          - { AMDGPU_FAMILIES: gfx120X-all }
```

Comment:
> As a side effect this also enables building PyTorch wheels for gfx950.

What about gfx1151? Does this fix https://github.com/ROCm/TheRock/issues/923 ?

Reply:
It indeed does! See https://github.com/ROCm/TheRock/actions/runs/16166448085.